api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java

```diff
@@ -326,10 +326,128 @@ public CohortDefinition getPatientsWhoSuspendedTreatment() {
   }
 
   /** @return B8: Number of dead patients during the current month */
-  public CohortDefinition getPatientsWhoDied() {
-    EncounterWithCodedObsCohortDefinition cd = getStateOfStayCohort();
-    cd.addIncludeCodedValue(hivMetadata.getPatientHasDiedConcept());
-    cd.setName("Number of dead patients during the current month");
+  public CohortDefinition getPatientsWhoDied(Boolean data1) {
```

Comment:
Its hard to make sense of the variable `data1`, please use a more suggestive name.

Reply:
Done.